src/audits/manifest/short-name-length.js

```diff
@@ -50,14 +50,17 @@ class ManifestShortNameLength extends Audit {
     let debugString;
     const manifest = artifacts.manifest.value;
     const suggestedLength = 12;
+    const shortNameValue = manifest && manifest.short_name && manifest.short_name.value;
+    const nameValue = manifest && manifest.name && manifest.name.value;
+    const manifestValue = shortNameValue || nameValue;
 
-    if (manifest && manifest.short_name && manifest.short_name.value) {
+    if (manifestValue) {
       // Historically, Chrome recommended 12 chars as the maximum length to prevent truncation.
```

Comment:
Can you add a comment here that explains that name is used if short_name not present (hopefully with a link to the docs/source on that?)